hw/drivers/nimble/nrf52/src/ble_phy.c

```diff
@@ -1824,6 +1833,28 @@ ble_phy_resolv_list_disable(void)
 }
 #endif
 
+#if MYNEWT_VAL(BLE_LL_DIRECT_TEST_MODE) == 1
+void ble_phy_enable_dtm(void)
+{
+    /* Disable whitening */
+    NRF_RADIO->PCNF1 = NRF_MAXLEN |
+                       (RADIO_PCNF1_ENDIAN_Little <<  RADIO_PCNF1_ENDIAN_Pos) |
+                       (NRF_BALEN << RADIO_PCNF1_BALEN_Pos);
```

Comment:
we'll now have PCNF1 set in 3 different places in code now - these will need to be keep in sync, so probably it's better just toggle single bit: NRF_RADIO->PCNF1 &= ~RADIO_PCNF1_WHITEEN_Msk;